bricks/config.rb

```diff
@@ -93,7 +93,7 @@
 # or, to map host folders to guest folders of the same name,
 # $shared_folders = Hash[*['/home/foo/app1', '/home/foo/app2'].map{|d| [d, d]}.flatten]
 #$shared_folders = {}
-$shared_folders = {'/var/log/amobee' => '/var/log/amobee', '/etc/amobee' => '/etc/amobee', '/var/amobee' => '/var/amobee' }
+$shared_folders = {'/var/log/amobee' => '/var/log/amobee', '/etc/amobee' => '/etc/amobee'}
```

Comment:
I thought we still use `/var/amobee` to store the pod and pv related stuff? This removes mapping that into the vm?

Reply:
This will remove the NFS mount from the localhost into the guest machine. Instead we mount the .vdi image into the guest